research/conditional/train/configs/baselines/gpt_medium.yaml

```diff
@@ -0,0 +1,40 @@
+# grid args
+runner: "research.conditional.train.cc_train"
+time: "0-02:00:00"
+n_gpus: 1
+interactive_debug: false
+
+# train params
+params:
+  # model
+  model_type: "gpt"
+  dataset_type: "c4"
+  dmodel: 512
+  dff: 2048
+  n_att_heads: 8
+  n_blocks: 8
+  cutoff: 128
+  batch_size: 256
+  loss_checkpoint_chungs: 64
```

Comment:
How did this number come up? is it related to the vocab size somehow?

Reply:
This number should be around vocab_size / d_model.  64 is the closest that also divides the batch size.

Reply:
I understand, thnanks